src/morphosource/download.py

```diff
@@ -0,0 +1,54 @@
+import requests
+from requests.exceptions import HTTPError
+from morphosource.config import Endpoints
+from morphosource.exceptions import RestrictedDownloadError
+
+RESTRICTED_DOWNLOAD_MSG = """You do not have authorization to download this restricted media.
+Please visit https://www.morphosource.org and request download permission for media id:"""
+
+
+class DownloadConfig(object):
+    def __init__(self, api_key, use_statement, use_categories=None, use_category_other=None):
+        self.api_key = api_key
+        self.use_statement = use_statement
+        self.use_categories = use_categories
+        self.use_category_other = use_category_other
+        if not self.use_categories and not self.use_category_other:
+            raise ValueError("Either use_categories or use_category_other must have a value.")
+
+
+class DownloadVisibility(object):
+    # In the MorphoSource UI this is called Publication
+    OPEN = "Open Download"
+    RESTRICTED = "Restricted Download"
+
+
+def get_download_media_zip_url(media_id, download_config):
+    url = f"{Endpoints.DOWNLOAD}/{media_id}"
+    data = {"use_statement": download_config.use_statement, "agreements_accepted": True}
```

Comment:
Should we have a note somewhere that by using this package you are consenting to the MorphoSource user agreements (and link to it or something)?

Reply:
Perhaps we should put that at the beginning of the README. I can't seem to find a good location for the various agreements outside of trying to download specific files. There are different Usage Agreements for the various license types.

Reply:
Could just add this [user agreement ](https://www.morphosource.org/terms) right above the TOC? It seems to cover the broad contents.

Reply:
I added the following note combining the two suggestions:
https://github.com/Imageomics/pyMorphoSource/blob/bbca1d6bfe34b6d0f3e71673ac1d19e069365547/README.md?plain=1#L1-L5